gfx/GFX_Renderer.h

```diff
@@ -36,9 +36,9 @@ namespace gfx
 		void renderPresent() const;
```

Comment:
Preferred structure to keep gfx free from SDL contamination

project gfx
```
    class Renderer {
        virtual void renderBox(..) = 0;
        virtual void renderText(…) = 0;
        // etc

    protected:
        Renderer() = default;
    };

```
project paint 
```
    class SDLRenderer : public gfx::Renderer {
    public:
        SDLRenderer(SDL_Renderer * sdl)
        : sdl_(sdl)
       {
       }
       
       virtual void renderBox(...) override;
       virtual void renderBox(...) override;
       virtual void renderBox(...) override;

    private:
        SDL_Renderer * sdlRenderer_;
    };
    

```

Reply:
done